wwivutil/messages/messages.cpp

```diff
@@ -164,15 +164,16 @@ class PostMessageCommand final : public BaseMessagesSubCommand {
   PostMessageCommand() : BaseMessagesSubCommand("post", "Posts a new message.") {}
 
   bool AddSubCommands() override {
-    add_argument({"title", "message sub name to post on", ""});
-    add_argument({"from", "message sub name to post on", ""});
-    add_argument({"from_usernum", "message sub name to post on", ""});
-    add_argument({"to", "message sub name to post on", ""});
-    add_argument({"date", "message sub name to post on", ""});
-    add_argument({"in_reply_to", "message sub name to post on", ""});
+    add_argument({"title", "title of post", ""});
```

Comment:
oh thanks!

Reply:
i need clarification on what in_reply_to refers to though. what are the expected possible values here?

Reply:
from message_area_wwiv.cpp

    } else if (starts_with(line, "RE:")) {
      r.in_reply_to = StringTrim(line.substr(3));
   ....

It's just the "RE: XXXXX" line text without the "RE: "